docs/framework/react/start/getting-started.md

```diff
@@ -241,7 +246,7 @@ Now that we have the basic templating setup, we can write our first route. This
 
 ```tsx
 // app/routes/index.tsx
-import * as fs from 'fs'
+import * as fs from 'node:fs'
```

Comment:
that's the only thing I would take over, please remove the others

Reply:
better to keep this in, since Bun and Deno want the node prefix